.percy.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+snapshot:
+  widths:
+    - 1280
```

Comment:
@bclark-p44 I limited this to one width to save on screenshots. We can easily add `375px` to cover mobile.

Reply:
Yeah we will be adding our adaptive sizes to the design system soon.